.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -1,22 +1,42 @@
 ---
-# This is a gha updates automaticaly the notebook controller images. Can be run on demand before a new release
-name: Update Notebook Controller Images With Latest Commit ID
-on:  # yamllint disable-line rule:truthy
+# This workflow automatically updates the notebook controllers' image tags
+name: Update Notebook Controllers' Image Tags
+on:
   workflow_dispatch:
     inputs:
       branch-name:
-        description: "Provide name of the branch, ex: v1.9-branch"
+        description: "Provide name of the branch, used to commit changes"
         required: true
-        default: "v1.9-branch"
+        default: "main"
```

Comment:
why default to `main` when we aren't going to be running this against main in the future?

* https://github.com/opendatahub-io/kubeflow/pull/468

Reply:
I have to remove from this PR, the same change that you do on the #468